pkg/cmd/server/server.go

```diff
@@ -501,6 +509,13 @@ func (s *server) runControllers(config *api.Config) error {
 		s.logger,
 	)
 
+	go func() {
+		http.Handle("/metrics", prometheus.Handler())
```

Comment:
I would prefer not to use package-level http handler registration. Would you mind creating an `http.ServeMux`, registering the handler with that, and then passing the `ServeMux` as the 2nd parameter to `http.ListenAndServe`?